update unverified sender treatment support article (#1168)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/office-365-security/unverified-sender-feature.md

# Unverified Sender

> [!NOTE]

> > These updates are rolling out now, and might not be available yet for all users. This feature is supported for Enterprise outlook.com and Enterprise Outlook Win32 desktop users. It is not currently available for consumer office 365 users.

Reviewer: ```suggestion
> These updates are rolling out now, and might not be available yet for all users. This feature is supported for Enterprise Outlook.com and Enterprise Outlook Win32 desktop users. It is not currently available for consumer Office 365 users.
```

> To prevent phishing messages from reaching your mailbox, Office 365 verifies that the sender is who they say they are and mark suspicious messages as junk email.

Reviewer: ```suggestion
To prevent phishing messages from reaching your mailbox, Office 365 verifies that senders are who they say they are and mark suspicious messages as junk email.
```

> [!IMPORTANT]
> When a message is marked as a phishing scam, Outlook displays a warning at the top of the page, but any links in the message can still be opened.

## How can I identify a suspicious message in my inbox?

Outlook shows indicators when the sender of a message either can't be identified or their identity is different from what you see in the From address.

## You see a '?' in the sender image

> When Office 365 can't verify the identity of the sender using email authentication techniques, they display a '?' in the sender photo.

Reviewer: ```suggestion
When Office 365 can't verify the identity of the sender using email authentication techniques, a '?' is displayed in the sender image.
```

### What criteria does Outlook.com and Outlook Win32 desktop use to add the '?' and the 'via' properties?

For the '?' in the sender image:  Outlook.com requires that the message pass either SPF or DKIM authentication and receive either a dmarc pass, or a composite authentication pass from Office 365 Spoof Intelligence. For details, see [Set up SPF in Office 365 to help prevent spoofing](set-up-spf-in-office-365-to-help-prevent-spoofing.md) and [Use DKIM to validate outbound email sent from your custom domain in Office 365](use-dkim-to-validate-outbound-email.md).

For the via tag: If the domain in the From address is different from the domain in the DKIM signature or the SMTP MAIL FROM, Outlook.com displays the domain in one of those two fields (preferring the DKIM signature).

### How do I remove the '?' without utilizing the Spoof Intelligence spoof allow list?

For the '?' in the sender image: As a sender, you should authenticate your message with either SPF or DKIM.

For the via tag: As a sender, you should ensure that either the domain in the DKIM signature or the SMTP MAIL FROM is the same as, or is a subdomain of, the domain in the From address.

> ### Does Outlook.com and Outlook Win32 on the web show this for every message that doesn't pass authentication?

Reviewer: ```suggestion
### Do Outlook.com and Outlook Win32 desktop show this for every message that doesn't pass authentication?
```
